pkg/controller/backup_session.go

```diff
@@ -82,15 +82,21 @@ func (c *StashController) runBackupSessionProcessor(key string) error {
 	backupSession := obj.(*api_v1beta1.BackupSession)
 	glog.Infof("Sync/Add/Update for BackupSession %s", backupSession.GetName())
 
+	if backupSession.Status.Phase == api_v1beta1.BackupSessionFailed ||
+		backupSession.Status.Phase == api_v1beta1.BackupSessionSucceeded {
+		log.Infof("Skipping processing BackupSession %s/%s. Reason: phase is %q.", backupSession.Namespace, backupSession.Name, backupSession.Status.Phase)
+		return nil
+	}
+
 	// check weather backup session is completed or running and set it's phase accordingly
 	phase, err := c.getBackupSessionPhase(backupSession)
```

Comment:
What happens to this error?

Reply:
It's not error of operator itself. It is the error that failed backup for a host that we wrote in `backupOutput`. So, we just used this error to write failure event.